src/broker/index.js

```diff
@@ -504,6 +504,26 @@ module.exports = class Broker {
     return await this.connection.send(createTopics({ topics, validateOnly, timeout }))
   }
 
+  /**
+   * @public
+   * @param {Array} topicPartitions e.g:
```

Comment:
This is missing the `count` argument., and in the documentation `newAssignments` is referred to as `assignments`. I personally prefer `assignments`.

Reply:
Will fix it. I used assignments in the requests, so it should match here